### PR TITLE
updated pojos

### PR DESCRIPTION
**Pull request by rjindal365**

### CODE REVIEW

File: src/main/java/net/media/openrtb3/Ad.java [hunk 1]
```diff
@@ -50,11 +50,11 @@
 
   public Ad() {}
 
-  public @NotNull String getId() {
```
Comment (shivabhalla) [inline src/main/java/net/media/openrtb3/Ad.java:53]: Check the purpose for notNull at getter/setter

File: src/main/java/net/media/openrtb3/Seatbid.java [hunk 2]
```diff
@@ -52,11 +52,11 @@ public void set_package(Integer _package) {
     this._package = _package;
   }
 
-  public @NotNull @Valid Collection<Bid> getBid() {
+  public @NotEmpty @Valid Collection<Bid> getBid() {
     return this.bid;
   }
 
-  public void setBid(@NotNull @Valid Collection<Bid> bid) {
+  public void setBid(@NotEmpty @Valid Collection<Bid> bid) {
```
Comment (shivabhalla) [inline src/main/java/net/media/openrtb3/Seatbid.java:59]: Check this in openRtb doc if it allows empty bid array. As bidders can send this.

File: src/main/java/net/media/utils/validator/Ortb3EventObjectValidator.java [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+import net.media.openrtb3.Event;
+import org.apache.commons.lang3.StringUtils;
+
+import javax.validation.Constraint;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import javax.validation.Payload;
+import java.lang.annotation.*;
+
+/** Created by rohit.ji on 23/04/19. */
```
Comment (shivabhalla) [inline src/main/java/net/media/utils/validator/Ortb3EventObjectValidator.java]: Write docs as discussed

File: src/main/java/net/media/utils/validator/Ortb3EventObjectValidator.java [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    Class<? extends Payload>[] payload() default {};
+
+    class Validator implements ConstraintValidator<Ortb3EventObjectValidator, Event> {
+        @Override
+        public boolean isValid(Event event, ConstraintValidatorContext context) {
+            if (event == null || event.getMethod() == null) {
+                return true;
+            }
+            int method = event.getMethod();
+            if (1 <= method && method <= 2) {
```
Comment (CoderOnAHigh) [inline src/main/java/net/media/utils/validator/Ortb3EventObjectValidator.java]: method == 1 || method == 2

**Pull request merged**